src/composables/useGetMessages.ts

```diff
@@ -539,6 +556,70 @@ export function useGetMessagesProvider() {
 		}, 500)
 	}
 
+	/**
+	 * Try to start chat relay
+	 *
+	 * @param options
+	 * @param options.force - to skip end reached check when it is guaranteed
+	 */
+	function tryChatRelay(options?: { force: boolean }) {
+		if (chatRelaySupported && (isChatEndReached.value || options?.force)) {
+			startChatRelay()
+		}
+	}
+
+	/**
+	 * Check whether chat relay is supported
+	 *
+	 * @param features
+	 */
+	function checkChatRelaySupport(features: string[]) {
+		if (experimentalChatRelay && features.includes('chat-relay')) {
+			chatRelaySupported = true
+		}
```

Comment:
In case we connect to another HPB/server that doesn't have this feature
```suggestion
		} else {
			chatRelaySupported = false
		}
```